### DISCUSSION TITLE
Any screenrecorder like ssrc

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#771] [Q&A]
I found wf-recorder which can be used in a script.
Any suggestions or scripts that I could look at?

**Comment by kRHYME7** [comment]
hmm, have you tried using this? 
https://github.com/russelltg/wl-screenrec

I remembered writing something for this. I can rewrite it together with the screenshot utility. 
wl-screenrec said they are better than wf-recorder too.

**Comment by se7en-x230** [comment]
Looks "usable" 

I would appreciate a simple way to record via a script/keybinding

**Comment by kRHYME7** [comment]
I think I will add this to the CLI. https://github.com/kRHYME7/Hyprdots-ctl. Also maybe you can add an issue here so that we can keep track of something.
